src/TileManager.js

```diff
@@ -78,13 +78,14 @@ export async function getStorageInfo(urlTemplate) {
  * @param {string} tileUrl
  * @return {Promise<blob>}
  */
-export async function downloadTile(tileUrl) {
-  return fetch(tileUrl).then((response) => {
+ export async function downloadTile(tileUrl) {
+  return fetch(tileUrl).then(response => {
     if (!response.ok) {
-      throw new Error(`Request failed with status ${response.statusText}`);
-    }
+      throw new Error("Request failed with status ".concat(response.statusText));
```

Comment:
What is the reason for this change? Also i am not sure, a concat function exists (it exists as method on various objects)

Reply:
There is no reason. A mistaking on my side